app-frontend/src/app/components/projects/projectItem/projectItem.html

```diff
@@ -1,14 +1,51 @@
 <div class="panel-body">
-  <div class="tag-container">
-    <h4 class="project-title">{{$ctrl.project.name}}</h4>
-    <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <a href ui-sref="projects.detail({projectid: $ctrl.project.id})" class="h4">
+    <h4 class="project-title" data-title="{{$ctrl.project.name}}">{{$ctrl.project.name}}</h4>
+  </a>
+  <div class="row">
+    <div class="column-12">
+
+      <!-- IF project has no scenes -->
+      <div class="project-preview-container placeholder" 
+           ng-if="!$ctrl.project.scenes && $ctrl.status !== 'CURRENT'"
+           style="background-image: url({{$ctrl.projectPlaceholder}})">
+        <span>This project doesn't have any scenes yet</span>
+      </div>
+
+       <!-- IF no ingested scenes: -->
```

Comment:
That's a good question. I believe the way we handle it now is that if there are any un-ingested scenes that we don't display a thumbnail. So to answer your question, the project would show the placeholder once the un-ingested scene was added. I don't recall if we _can_ or _can't_ generate a thumbnail if there are un-ingested scenes within a project though.